examples/consumer-and-issuer/src/isle.provider.tsx

```diff
@@ -111,6 +112,50 @@ export function IsleProvider({ children, containerId }: IsleProviderProps) {
             return externalSigner;
           },
         };
+      } else if (walletType === "stellar") {
+        walletInfo = {
+          address: walletAddress,
+          publicKey: walletPublicKey,
+          signMethod: async (message: string) => {
+            // Encode the message as base64 (stellarKit expects this)
+            const messageBase64 = Buffer.from(message).toString("base64");
```

Comment:
If you will use freigheter api directly, this is not required... the big mess with base64 is done by stellarkit library :( but since we (I hope) want to extract this to library, let's keep it like this.